collectors/icdcm/collector.py

```diff
@@ -59,5 +61,5 @@ def collect(conf, conn):
             if not count % 100:
                 logger.info('Collected %s "%s" conditions', count, record.table)
 
-        except Exception as exception:
-            logger.exception(repr(exception), exc_info=True)
+        except:
+            base.config.SENTRY.captureException()
```

Comment:
Yes, they do capture the traceback. In general, I added the basic info for each exception logging where it made sense, or logged the exception only when not. I might have missed data in some places, and would be happy to ad. Otherwise we can add when we find them.